skimage/restoration/_denoise.py

```diff
@@ -56,8 +56,15 @@ def denoise_bilateral(image, win_size=5, sigma_range=None, sigma_spatial=1,
 
     """
     mode = _mode_deprecations(mode)
-    return _denoise_bilateral(image, win_size, sigma_range, sigma_spatial,
-                              bins, mode, cval)
+
+    offset = np.min(image)
+    offset_img = image - offset
+    if not offset_img.any():
+        # if all value identical, returns the same image
+        return image
+    else:
+        return _denoise_bilateral(offset_img+1, win_size, sigma_range, sigma_spatial,
```

Comment:
well, it was related to a previous correction (commit 8d6ba04),
now, with the offset in the python wrapper it is indeed not needed anymore, so I reverted the .pyx to its initial state and removed the +1/-1